src/compliance/compliance-requirements.controller.spec.ts

```diff
@@ -99,4 +101,67 @@ describe('ComplianceRequirementsController', () => {
       expect(result).toEqual(response);
     });
   });
+
+  describe('deleteRequirement', () => {
+    it('should accept TransactionBaseDto and compliance requirement id and delete Asset Compliance rule', async () => {
```

Comment:
```suggestion
    it('should accept TransactionBaseDto and compliance requirement ID and delete the corresponding Asset Compliance rule for the given ticker', async () => {
```
If you are trying to give a full idea of the method, its better to mention about the ticker as well